certora/specs/Initializable.spec

```diff
@@ -0,0 +1,138 @@
+import "helpers.spec"
+
+methods {
+    // initialize, reinitialize, disable
+    initialize()
+    reinitialize(uint8)
+    disable()
+
+    // view
+    version()      returns uint8 envfree
+    initializing() returns bool  envfree
```

Comment:
There one are view. The above are state changing.

The above used to be payable, which AFAIK prevents envfree. I had to rewrite them as nonpayable, not sure if they can be made envfree now. That might be possible since msg.sender is irrelevant.